server/src/main.rs

```diff
@@ -37,12 +37,24 @@ async fn main() -> Result<(), BoxedStdError> {
 
     let args = cli::parse();
 
-    let data = fs::read_to_string(args.path)
-        .unwrap_or_else(|error| panic!("Could not read the startup state.\nError: {error}"));
+    let data = fs::read_to_string(args.path).unwrap_or_else(|error| {
+        use std::io::ErrorKind as EK;
```

Comment:
I don't really think this helps with the readability.

Reply:
Even as a programmer I have to search for the error cause in this message, there is a lot of noise included:
```text
thread 'main' panicked at 'Parsing start config failed with error missing field `workloads`', server/src/main.rs:52:33
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Using logging output also has some noise at the beginning (timestamp), but at least to me its easier to read:
```text
[2023-09-07T15:29:58Z ERROR ank_server] Parsing start configuration file failed with error: missing field `workloads`.
```
Originally I `eprintln`ed the text out (which would be consistent with most other applications) but decided to be consistent with the remaining ankaios code.

Reply:
Sorry for the confusion, I meant the aliasing of ErrorKind to EK.